addons/weather/functions/fnc_getMapData.sqf

```diff
@@ -239,3 +226,17 @@ if (_worldName in ["kunduz"]) exitWith {
         [0.04, 0.02, 0.05, 0.14, 0.19, 0.07, 0.10, 0.07]  // December
     ];
 };
+
+
+// Catches any "Winter" Map that hasnt been defined otherwise - this should stay at the end of the file
+// Values are not based on any RL reference since the snow terrain textures persists regardless the date
+private _cfgPath = (_cfgPath >> "RainParticles");
+if (
+    "winter" in _worldName || 
+    {"snow" in getText (_cfgPath >> "rainDropTexture")} ||
+    {!isNull (_cfgPath >> "snow") && { getNumber (_cfgPath2 >> "snow") != 0 }}
```

Comment:
If it doesn't exist, it will return `0` regardless, so not worth the `isNull` check.
```suggestion
    {getNumber (_cfgPath >> "snow") != 0}
```

Reply:
wasnt aware, thanks for letting me know!